src/helpers/schema.cjs

```diff
@@ -1,57 +1,122 @@
-const Ajv = require('ajv');
+const Ajv = require('ajv/dist/2019');
```

Comment:
We now use the [2019 JSON schema draft](https://json-schema.org/draft/2019-09/release-notes).